test/coverage/datamodel/UUriSerializerTest.cpp

```diff
@@ -23,16 +23,167 @@ class TestFixture : public testing::Test {
 
 	// Run once per execution of the test application.
 	// Used for setup of all tests. Has access to this instance.
-	TestFixture() = default;
-	~TestFixture() = default;
+	TestUUriSerializaerFixture() = default;
+	~TestUUriSerializaerFixture() = default;
 
 	// Run once per execution of the test application.
 	// Used only for global setup outside of tests.
 	static void SetUpTestSuite() {}
 	static void TearDownTestSuite() {}
 };
 
-// TODO replace
-TEST_F(TestFixture, SomeTestName) {}
+std::shared_ptr<uprotocol::v1::UUri> buildValidTestURI(
```

Comment:
Why does it need to be a shared_ptr?

Reply:
Done. Its not needed. I have changed it to local object.